server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java

```diff
@@ -460,15 +459,12 @@ public void run() {
     log.info("stop requested. exiting ... ");
   }
 
-  private ServletHolder getDefaultServlet() {
-    return new ServletHolder(new DefaultServlet() {
-      private static final long serialVersionUID = 1L;
-
-      @Override
-      public Resource getResource(String pathInContext) {
-        return Resource.newClassPathResource("/org/apache/accumulo/monitor" + pathInContext);
-      }
-    });
+  private ServletHolder getResourcesServlet() {
+    ServletHolder holder = new ServletHolder("resources", ResourceServlet.class);
+    holder.setInitParameter("dirAllowed", "false");
+    holder.setInitParameter("baseResource", Monitor.class.getClassLoader()
+        .getResource("org/apache/accumulo/monitor/resources").toExternalForm());
```

Comment:
Since the Monitor class is in the `org/apache/accumulo/monitor` package, this can probably be simplified to a relative resource:

```suggestion
    holder.setInitParameter("baseResource", Monitor.class.getResource("resources")
        .toExternalForm());
```

Using the previous `getClassLoader()` form, it should probably be an absolute resource, starting with `/`, in order to not make any assumptions about the system classloader's relative resource path resolution.

Also, I don't know if this configuration is actually correct and has equivalent functionality... I hope so... but would need to test it to see how it behaves at runtime, since I don't know enough about the ResourceServlet's behavior.